art/art.py

```diff
@@ -125,28 +125,30 @@ def help_func():
     print(CLI_HELP)
 
 
-def aprint(artname, number=1):
+def aprint(artname, number=1,space=1):
     """
     Print 1-line art.
 
     :param artname: artname
     :type artname : str
     :param number: number of repeats
     :type number: int
+    :param space: space between characters
+    :type space: int
     :return: None
     """
     try:
         if artname == "UnicodeEncodeError":
             raise UnicodeEncodeError(
                 'test', u"", 42, 43, 'test unicode-encode-error')
         result, artname = art(
-            artname=artname, number=number, __detailed_return=True)
+            artname=artname, number=number, __detailed_return=True,space=space)
         print(result)
     except UnicodeEncodeError:
         print(ART_ENVIRONMENT_WARNING.format(artname))
 
 
-def art(artname, number=1, __detailed_return=False):
+def art(artname, number=1, __detailed_return=False,space=1):
```

Comment:
change it to `def art(artname, number=1, __detailed_return=False, space=1):` (PEP8)